handlers/root.go

```diff
@@ -13,3 +15,19 @@ func Root(w http.ResponseWriter, r *http.Request) {
 func getStatus(w http.ResponseWriter, r *http.Request) {
 	RespondWithJSONIndent(w, http.StatusOK, status.Get())
 }
+
+func GetToken(w http.ResponseWriter, r *http.Request) {
+	u, _, ok := r.BasicAuth()
+	if !ok {
+		RespondWithJSONIndent(w, http.StatusInternalServerError, u)
+		return
+	}
+	fmt.Printf("Token " + u)
```

Comment:
This print shouldn't be a log.info if it is intentional print not debug line ?

Reply:
Removed